python/py-tensorflow-tensorboard/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tensorflow-tensorboard
+version             0.4.0rc3
+platforms           darwin
+license             Apache-2
+maintainers         @emcrisostomo openmaintainer
+
+description         TensorFlow helps the tensors flow
+long_description    TensorBoard is a suite of web applications for inspecting \
+                    and understanding your TensorFlow runs and graphs.
+
+homepage            https://github.com/tensorflow/tensorboard
+master_sites        https://pypi.python.org/packages/d5/fb/80df4eb3234c41beeeb3122b1966effbf08608fb80d2fa74bb72f8da9cb3
+
+if {${python.version} < 30} {
+    distname            tensorflow_tensorboard-${version}-py2-none-any
+} else {
+    distname            tensorflow_tensorboard-${version}-py3-none-any
+}
+
+extract.suffix      .whl
+extract.cmd         cp
+extract.pre_args
+extract.post_args   ${worksrcpath}
+extract.mkdir       yes
```

Comment:
This could all probably be avoided if you told pip to install the wheel from `${distpath}/${distfiles}` in destroot.args below.

Reply:
Ah ok. Can I then skip the extract phase with a simple:

``` extract {
}```

?

Reply:
Yes.

Reply:
Done. Thanks, it made the trick